pcdsdevices/sim/motor.py

```diff
@@ -0,0 +1,69 @@
+import time
+import threading
+
+from ophyd.device import Device, Component as Cpt
+from ophyd.positioner import SoftPositioner
+from ophyd.signal import AttributeSignal
+from ophyd.sim import SynAxis
+
+from pcdsdevices.mv_interface import FltMvInterface
+
+
+class SynAxisInterface(FltMvInterface, SynAxis):
```

Comment:
Too confusing just to refer to as `SynAxis`? `SynMotor`? Find this name kind of a drag ...

Reply:
I thought having a `SynMotor` and a `SimMotor` would be annoying. I thought that copying the `SynAxis` named would be annoying.

How about `SynMotor` and `SlowMotor`?

Reply:
I like `SynMotor` `SlowMotor`